bin/tagpreview.sh

```diff
@@ -0,0 +1,60 @@
+#!/usr/bin/env bash
+
+REVERSE="\x1b[7m"
+RESET="\x1b[m"
+
+if [ -z "$1" ]; then
+  echo "usage: $0 FILENAME:TAGFILE:EXCMD"
+  exit 1
+fi
+
+IFS=':' read -r FILE TAGFILE EXCMD <<< "$*"
+
+FILE="$(dirname "${TAGFILE}")/${FILE}"
+
+if [ ! -r "$FILE" ]; then
+  echo "File not found ${FILE}"
+  exit 1
+fi
+
+CENTER="$(vim -i NONE -u NONE -e -m -s "${FILE}" \
+              -c "set nomagic" \
+              -c "${EXCMD}" \
+              -c 'let l=line(".") | new | put =l | print | qa!')" || return
+
+START_LINE="$(( CENTER - FZF_PREVIEW_LINES / 2 ))"
+if (( START_LINE <= 0 )); then
+    START_LINE=1
+fi
+END_LINE="$(( START_LINE + FZF_PREVIEW_LINES - 1 ))"
+
+# Sometimes bat is installed as batcat.
+if command -v batcat > /dev/null; then
+  BATNAME="batcat"
+elif command -v bat > /dev/null; then
+  BATNAME="bat"
+fi
+
+if [ -z "$FZF_PREVIEW_COMMAND" ] && [ "${BATNAME:+x}" ]; then
+  ${BATNAME} --style="${BAT_STYLE:-numbers}" \
+             --color=always \
```

Comment:
I was able to apply manually this patch to my fzf.vim 

1) I did not include the changes from https://github.com/junegunn/fzf.vim/pull/1223/commits/9067bcc4bfbba128c6c969700049c16d7dbe3485
2) I added to the above code to make it work with my

```
--theme="GitHub" \
```

Reply:
`bat` will use the theme set in `$BAT_THEME` environment variable so we don't need to change this code.